app/models/json_schema.rb

```diff
@@ -3,7 +3,7 @@ class JsonSchema
   class << self
 
     def generate(json_template)
-      schema = JSON::SchemaGenerator.generate 'Claim for crown court defence - claim import', json_template
+      schema = File.read(File.join(Rails.root, 'config', 'claim_schema.json'))
```

Comment:
Ive raised a techdebt icket to look at this in more detail (https://www.pivotaltracker.com/story/show/113434493).  We need to take a decision as to whether or not to automatically generate the json schema as before or not, and then update the code accordingly.